### DISCUSSION TITLE
No comm with board

### DISCUSSION THREAD

**Discussion by magimat** [aqualinkd/AqualinkD#179] [General]
Hi, I am trying to connect to my basic board which is only a salt generator. Aquarite AQRXL-LS-CUL

I found and connected my RS485 usb dongle to the correct terminals inside the panel. Connected A on A, B on B and supplied 10v through 10V and GND terminals and also wired ground to the ground pin on my dongle so I'm pretty sure my wiring is ok, but still, I get absolutely no connection when I start Aqualinkd.

I didnt change anything in the config file and am wondering if I should adjust panel_type or device_id, but I have no idea what I am supposed to put in those

Aqualinkd seems to be running fine, just not connecting to the panel and the "remotely controlled" led on my panel never gets lit.

Any idea?

**Comment by magimat** [comment]
here is my rs485 dongle:  https://www.amazon.ca/gp/product/B081NBCJRS/ref=ppx_yo_dt_b_asin_title_o07_s00?ie=UTF8&th=1

**Comment by sfeakes** [comment]
AqualinkD can not control a SWG without a control panel.  So you can't just connect AqualinkD directly to a salt water generator.

**Comment by magimat** [comment]
I see, would your other repo AquapureD be a solution for my situation?

**Comment by sfeakes** [comment]
AquapureD can control a SWG directly.  But it's not maintained and has a few issues. I put it out there as a few people had asked for it and it's how I used to control my SWG. But the person that was going to maintain it never started, and I don't have the time.  I believe there are some other repo's that can do what you want like https://github.com/tagyoureit/nodejs-poolController

**Comment by magimat** [comment]
Thanks for getting back, I’m aware of nodejs-poolController, but I’m only looking for something to put an api or MQTT interface over my SWG because am I integrating my own overall pool automation myself through home assistant so I liked your solution more because it looked simpler to me, but if I can’t get aquapured to do what I want I will either look into nodejs-poolController or maybe just implement my own very basic jandy protocol integretion. I looked into it and for what I need (get salt ppm and adjust generation %) it really doesn’t look that complicated.

I’d look into your aquapured repo myself, but I really don’t like cpp, more of a java node guy :)


Anyway, thanks a lot for your work and your answers!
